src/diffpy/utils/diffraction_objects.py

```diff
@@ -319,7 +341,8 @@ def get_array_index(self, value, xtype=None):
 
         Returns
         -------
-        the index of the value in the array
+        list
```

Comment:
let's check.  Logically, this should return an int not a list.  Let's check and maybe update the function/tesnts, or understand why it returns a list and change the name.

Reply:
Yup, I created a separate issue for this since it's beyond the scope of this PR. I will look into this.

Reply:
https://github.com/diffpy/diffpy.utils/issues/279